std/traits.d

```diff
@@ -5346,7 +5346,9 @@ convertible to a string.
  */
 template isConvertibleToString(T)
```

Comment:
FWIW I don't see [any uses out in the wild](https://github.com/search?utf8=%E2%9C%93&q=isConvertibleToString+extension%3Ad+fork%3Afalse&type=Code&ref=searchresults) but I'll just document it unless I hear otherwise.